README.md

```diff
@@ -148,3 +148,9 @@ console.log(await adb.getPIDsByName('m.android.phone'));
 - uninstallApk
 - installFromDevicePath
 - install
+- setAutoRotationState
+- isScreenLandscape
+- isScreenPortrait
+- setOrientation
+- beLandscape
```

Comment:
I'm not a big fan of `beXXX` method names. There is nothing else in the codebase like that. I'd prefer just the `setOrientation` method, and the two orientations having constant values. Something like

```js
const orientations = {
  LANDSCAPE: 1,
  PORTRAIT: 0
}
```